### PR TITLE
Bumped ML sim chart version

### PR DESCRIPTION
**Pull request by msk-**

### CODE REVIEW

**Review by mdebarros — COMMENTED** [review]
Don't forget to update Mojaloop/Chart.yaml to reflect the application change.

**Comment by msk-** [comment]
> Don't forget to update Mojaloop/Chart.yaml to reflect the application change.

That. Was. Fast.

Hah. Thanks. I think `mojaloop/requirements.yaml` needed updating, but not `Chart.yaml`. Did I miss anything else?

**Comment by elnyry-sam-k** [comment]
@msk- , about to merge this, FYI

**Comment by msk-** [comment]
> @msk- , about to merge this, FYI

@elnyry-sam-k thanks. Bump!

File: mojaloop-simulator/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 description: "Helm Chart for the Mojaloop (SDK based) Simulator - mojaloop-simulator: v11.2.1; sdk-scheme-adapter: v11.3.0"
 name: mojaloop-simulator
-version: 11.0.0
-appVersion: "11.0.0"
+version: 11.0.1
+appVersion: "11.0.1"
```
Comment (elnyry-sam-k) [inline mojaloop-simulator/Chart.yaml:5]:
```suggestion
appVersion: "11.2.1"
```

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**